traefik/Changelog.md

```diff
@@ -1,5 +1,55 @@
 # Change Log
 
+## 19.0.4 
+
+**Release date:** 2022-11-07
+
+![AppVersion: 2.9.4](https://img.shields.io/static/v1?label=AppVersion&message=2.9.4&color=success&logo=)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+* 🔧 Adds more supported settings on ServiceMonitor 
```

Comment:
Can you mention the change from scrapeInterval to Interval and it does now follow convention of the CRD itself?

Reply:
Sure. I updated the PR.